tests/models/test_gemini.py

```diff
@@ -526,14 +529,22 @@ async def test_request_structured_response(get_gemini_client: GetGeminiClient):
 async def test_request_tool_call(get_gemini_client: GetGeminiClient):
     responses = [
         gemini_response(
-            _content_model_response(ModelResponse(parts=[ToolCallPart('get_location', {'loc_name': 'San Fransisco'})]))
+            _content_model_response(
+                ModelResponse(
+                    parts=[
+                        ToolCallPart(
+                            'get_location', {'loc_name': 'San Fransisco'}, tool_call_id=generate_tool_call_id()
+                        )
+                    ]
+                )
+            )
         ),
         gemini_response(
             _content_model_response(
                 ModelResponse(
                     parts=[
-                        ToolCallPart('get_location', {'loc_name': 'London'}),
-                        ToolCallPart('get_location', {'loc_name': 'New York'}),
+                        ToolCallPart('get_location', {'loc_name': 'London'}, tool_call_id=generate_tool_call_id()),
+                        ToolCallPart('get_location', {'loc_name': 'New York'}, tool_call_id=generate_tool_call_id()),
```

Comment:
```suggestion
                        ToolCallPart('get_location', {'loc_name': 'London'}),
                        ToolCallPart('get_location', {'loc_name': 'New York'}),
```
I think it's set by the field factory.